lightning/src/ln/features.rs

```diff
@@ -126,6 +126,13 @@ impl ChannelFeatures {
 			mark: PhantomData,
 		}
 	}
+
+	/// Takes the flags that we know how to interpret in an init-context features that are also
+	/// relevant in a channel-context features and creates a channel-context features from them.
+	pub(crate) fn relevant_init_flags_to_channel(_init_ctx: &InitFeatures) -> Self {
```

Comment:
Hmm, at least personally I really hate From unless its super, duper, absolutely clear, and documentation describing exactly *what* it does would be wholly redundant. I don't think thats true here cause we're only copying over known flags. I changed it to with_known_relevant_init_flags, so callsites read ChannelFeatures::with_known_relevant_init_flags, which I think is pretty clear.

Reply:
Should this be "supported" rather than "known" features? I'd consider dropping "relevant" as that should be implied by the fact that it returns `ChannelFeatures`.